### DISCUSSION TITLE
Using wdio mock on remote selenium grid

### DISCUSSION THREAD

**Discussion by pau22** [webdriverio/webdriverio#13973] [Q&A] [answered]
I was going through the documentation of the mocks and want to confirm if is possible to run the automation tests using wdio mocks in a remote Selenium Grid? If it's possible can you help me with any wiki or example to implement it?

**Comment by christian-bromann** [comment, marked as answer]
> want to confirm if is possible to run the automation tests using wdio mocks in a remote Selenium Grid?

With the latest Selenium Grid that supports WebDriver Bidi , it should support using mocks in WebdriverIO.

> If it's possible can you help me with any wiki or example to implement it?

Aside from the references we have [in the docs](https://webdriver.io/docs/mocksandspies/#specifying-custom-responses) what type of examples or further documentation are you looking for?

**Reply by pau22** [reply]
Thanks for the response @christian-bromann . I'll take a look to docs that you mentioned. 
Appreciated it.
